cli/src/execDocker/docker.js

```diff
@@ -86,7 +86,7 @@ export async function tagDockerImage({ image, version }) {
 
 // Function to push a Docker image
 export async function pushDockerImage({ image, version }) {
-  // TODO Probably no need to ask again for dockerHubUsername, we have it in idapp.config.json
+  // TODO Probably no need to ask again for dockerHubUsername, we have it in CONFIG_FILE
```

Comment:
I think we can remove this outdated TODO comment.
There is some logic in `askForDockerhubUsername()` to get username from config file if we have it.

Reply:
350171e6149d82d8cd7e32cd24665f8c70a7c9d1